src/main/java/com/hotels/bdp/circustrain/bigquery/extraction/container/UpdatePartitionSchemaAction.java

```diff
@@ -28,22 +29,26 @@
   private final Storage storage;
   private final ExtractionUri extractionUri;
   private final SchemaExtractor schemaExtractor;
+  private final Table sourceTableAsHive;
```

Comment:
`sourceTableAsHive`? why not just call it `table` like we've done with `partition`?

Reply:
It was named like this in the HivePartitionGenerator class already so I kept it consistent. But I can rename it to make it read easier

Reply:
Ah OK, consistency is good but `sourceTableAsHive` doesn't "read" that well IMHO, what do others think? I guess it's trying to say "This is a source Hive table in memory" as opposed to a "real" Hive table?